imgui-SFML.cpp

```diff
@@ -549,6 +549,15 @@ void SetLStickYAxis(sf::Joystick::Axis lStickYAxis, bool inverted) {
 }  // end of namespace SFML
 
 /////////////// Image Overloads
+void Image(const sf::Sprite& sprite) {
+    sf::FloatRect bounds = sprite.getLocalBounds();
+    sf::Vector2f scale = sprite.getScale();
+    sf::Color tintColor = sprite.getColor();
+    sf::Color borderColor = sf::Color::Transparent;
```

Comment:
No need to create a variable and pass it, as it's a default argument for border color in an overload